peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -79,6 +71,13 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                     win_height = int(win_width // aspect_ratio)
                     cv2.resizeWindow(self.window_name, win_width, win_height)
 
+        elif inputs["filename"] is not self.current_filename:
```

Comment:
Use `==` or `!=` for string equality checking. `is` is checking for reference.

Reply:
I originally wanted to take advantage of the speed gain from string interning, but I realized the rule for string interning is not really consistent in Python. I have changed to `==`.